project.clj

```diff
@@ -1,70 +1,43 @@
-(def tk-version "1.4.0")
-(def ks-version "1.3.0")
-
-
-(defproject puppetlabs/trapperkeeper-status "0.6.1-SNAPSHOT"
+(defproject puppetlabs/trapperkeeper-status "0.7.0-SNAPSHOT"
   :description "A trapperkeeper service for getting the status of other trapperkeeper services."
   :url "https://github.com/puppetlabs/trapperkeeper-status"
   :license {:name "Apache License, Version 2.0"
             :url "http://www.apache.org/licenses/LICENSE-2.0"}
 
-  :pedantic? :abort
-
-  :exclusions [org.clojure/clojure]
+  :min-lein-version "2.7.1"
 
-  :dependencies [[org.clojure/clojure "1.7.0"]
+  :parent-project {:coords [puppetlabs/clj-parent "0.2.5"]
+                   :inherit [:managed-dependencies]}
 
-                 ;; Dependencies which resolve version conflicts via
-                 ;; :pedantic? :abort in transitive dependencies
-                 [clj-time "0.11.0"]
-                 [ring "1.4.0"]
-                 [commons-codec "1.10"]
-                 [org.clojure/tools.macro "0.1.5"]
-                 [org.clojure/tools.logging "0.3.1"]
-                 [org.clojure/tools.reader "1.0.0-alpha1"]
-                 [org.clojure/tools.trace "0.7.9"]
-                 [org.slf4j/slf4j-api "1.7.13"]
-                 ;; end list of version conflict resolution dependencies
+  :pedantic? :abort
 
-                 [cheshire "5.6.1"]
-                 [prismatic/schema "1.1.1"]
-                 ;; ring-defaults brings in a bad, old version of the servlet-api, which
-                 ;; now has a new artifact name (javax.servlet/javax.servlet-api).  If we
-                 ;; don't exclude the old one here, they'll both be brought in, and consumers
-                 ;; will be subject to the whims of which one shows up on the classpath first.
-                 ;; thus, we need to use exclusions here, even though we'd normally resolve
-                 ;; this type of thing by just specifying a fixed dependency version.
-                 [ring/ring-defaults "0.2.0" :exclusions [javax.servlet/servlet-api]]
+  :exclusions [org.clojure/clojure]
 
-                 [slingshot "0.12.2"]
-                 [trptcolin/versioneer "0.2.0"]
-                 [org.clojure/java.jmx "0.3.1"]
+  :dependencies [[org.clojure/clojure]
 
-                 [puppetlabs/kitchensink ~ks-version]
-                 [puppetlabs/trapperkeeper ~tk-version]
-                 [puppetlabs/trapperkeeper-scheduler "0.0.1"]
-                 [puppetlabs/ring-middleware "1.0.0"]
-                 [puppetlabs/comidi "0.3.1"]
-                 [puppetlabs/i18n "0.4.3"]]
+                 [cheshire]
+                 [slingshot]
+                 [prismatic/schema]
+                 [trptcolin/versioneer]
+                 [ring/ring-defaults]
```

Comment:
Do you think we should list the `:exclusions` for `servlet-api` here or not.  I did in my [trapperkeeper-metrics PR](https://github.com/puppetlabs/trapperkeeper-metrics/pull/38) out of paranoia but it may not be necessary.